modules/ROOT/pages/showcase-apps/metrics.adoc

```diff
@@ -1,2 +1,17 @@
 include::{partialsdir}/attributes.adoc[]
 
+= {metrics-service}
+
+[discrete]
+== Prerequisites
+
+* [x] You have setup the services as described in xref:showcase-apps/service-setup.adoc[Service Setup].
```

Comment:
Shouldn't this link directly to the metrics setup guide?

Reply:
Same reasoning as above.